.github/workflows/cypress.yml

```
@@ -0,0 +1,108 @@
+       run: npm install
+    
+     - name: Setup .env
+       run: |
+         pwd && cp frontend/.env.example frontend/.env
+         echo $REACT_APP_TALAWA_URL
+         curl -s -X POST http://127.0.0.1:4000/graphql -H "Content-Type: application/json" -d '{"query":"{__typename}"}' 2>/dev/null
+    
+     - name: Run Cypress Tests with Dev Server
+       uses: cypress-io/github-action@v6
+       with:
+         working-directory: frontend
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 8 but found 9

(indentation)